### PR TITLE
Consolidate learning resources

### PR DESCRIPTION
**Pull request by U8NWXD**

### CODE REVIEW

File: Contributing-code-to-Oppia.md [hunk 1]
```diff
@@ -53,21 +53,7 @@ TODO(#4): Add link to IDE tutorial.
...
-
-  * For Angular 2+, consider checking out the official documentation [here](https://angular.io/docs), which also includes a [tutorial](https://angular.io/tutorial).
-
-* If you are new to HTML, some tutorials include [Mozilla's guide](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML) (which includes some practice assessments), as well as [this tutorial for beginners](http://htmldog.com/guides/html/beginner/).
-
-* We also have some backend (Python) projects available, but not as many, so we'd strongly recommend learning AngularJS if you have the opportunity and inclination -- otherwise, the range of projects you can take up will be more limited. That said, let us know at welcome@oppia.org if you'd like suggestions for non-frontend projects, and we'll do our best to help.
+See our [[page of learning resources|Learning-Resources]] for suggestions on how you can improve your development skills.
```
Comment (U8NWXD) [inline Contributing-code-to-Oppia.md:56]: Add motivation for why you might want to click on the link

File: Learning-Resources.md [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
+
+    * [Introduction to GitHub](https://lab.github.com/courses/introduction-to-github) covers how to use GitHub.
+
+  * More advanced:
+
+    * The other levels from [Learn Branching Git](https://learngitbranching.js.org/) cover git in more depth.
+    * You may find this [git visualizer](https://git-school.github.io/visualizing-git/) helpful for understanding more advanced git operations. It can be helpful for simple ones too!
+    * GitHub's [managing merge conflicts page](https://lab.github.com/courses/managing-merge-conflicts) explains how to address merge conflicts.
+
+* AngularJS (v1) and Angular 2+ are used for Oppia's frontend; we are currently in the process of migrating from the former to the latter.  Most of our open issues are in the frontend and require at least some knowledge of HTML, AngularJS/Angular and CSS, so knowledge of frontend coding will serve you well if you'd like to contribute to Oppia over the longer term.
```
Comment (U8NWXD) [inline Learning-Resources.md:22]: Split AngularJS and Angular. Note that AngularJS is not for new code. Explain that we are in the middle of migrating

Comment (U8NWXD) [reply Learning-Resources.md]: done

File: Contributing-code-to-Oppia.md (hunk 1 above)
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md]: done

**Comment by U8NWXD** [comment]
@DubeySandeep PTAL

**Review by DubeySandeep — APPROVED** [review]

File: Learning-Resources.md [hunk 3]
```diff
@@ -1,16 +1,40 @@
-### Beginners
+In general, it's easier to contribute to the Oppia codebase if you have some knowledge of git, as well as at least one of Python or AngularJS/Angular. You don't need to know all of this before you start, though! Many of our contributors have picked these skills up concurrently while tackling their first issues.
 
-The following tutorials are the bare minimum which will help you get started:
+That said, we strongly recommend that you be open to learning new things. If you need to brush up on some of the technologies used in Oppia, here are some resources that may help:
```
Comment (DubeySandeep) [inline Learning-Resources.md:3]: Should we add this page somewhere in the sidebar?

**Review by seanlip — APPROVED** [review]
LGTM, just had the same question as @DubeySandeep. Thanks!

File: Learning-Resources.md (hunk 3 above)
Comment (U8NWXD) [reply Learning-Resources.md:3]: done

**Pull request merged**